### DISCUSSION TITLE
How to remove multiple station locations for eQSL

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#3114] [Ideas / Feature Requests]
Now we need to create multiple station locations to ensure synchronization with eQSL for occasional (weekend) trips to the country house.
It would be great if it were possible to create a station location and have a table of linked eQSL profiles with their time periods.

**Comment by pd5wl** [comment]
For eQSL i just gave all my locations the same eQSL name. Works fine for me.
